README.md

```diff
@@ -37,56 +37,56 @@ $ pip install foliantcontrib.pandoc
 
 1. Create a new project:
 
-```shell
-$ foliant init
-Enter the project name: Hello Foliant
-✓ Generating Foliant project
-─────────────────────
-Project "Hello Foliant" created in /path/to/hello-foliant
-```
+    ```console
```

Comment:
Please revert, fenced code blocks are not supported in lists: https://foliant-docs.github.io/docs/backends/mkdocs/#fenced-code-is-not-rendered-in-list-items-or-blockquotes